src/stores/AccountDetailStore.ts

```diff
@@ -28,6 +32,19 @@ export default class AccountDetailStore {
     this.items = this.items.concat(txs);
   }
 
+  public startTxPolling = () => {
+    this.loadFromWallet();
+    this.getTransactionsInterval = setInterval(() => {
+      this.loadFromWallet();
```

Comment:
This is missing the whole logic of checking the block number and you are not calling the previous pages. It should follow this logic:
1. on interval tick, call RPC `getblockcount`
2. if new block num:
3. you need to call `getTransactions()` for each page you are on. eg. if you are on page 3, you need to call `getTransactions(1)` + `getTransactions(2)` + `getTransactions(3)` and then concatenate the data into one list then return

Reply:
Not sure if I am misunderstanding what is missing,  but the block count for my transactions does update, and it does update for all the pages that have been loaded with loadMore.
 I'm using this to do it by passing in this.pageNum:
@action public async loadFromWallet() {
    this.items = await this.load(this.pageNum);
  }